### PR TITLE
Work for IOS-220

### PR DESCRIPTION
**Pull request by SilviuOdobescu**

### CODE REVIEW

File: AlfrescoSDK [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 8fce3f9bef9951a0e1d79d534b3adbeb5c524f96
+Subproject commit 50b898a69448d9241200b6ff037a0047f852d692
```
Comment (mikeh) [inline AlfrescoSDK]: Could you revert this change please? The SDK update needs to happen in a separate commit for traceability.

Comment (SilviuOdobescu) [inline AlfrescoSDK]: Will try to revert back the change
